nexus/db-model/src/queries/region_allocation.rs

```diff
@@ -136,6 +122,8 @@ table! {
     }
 }
 
+// XXX These could do for some cleanup
+
```

Comment:
I can. The short version is just that there's so redundant allows that aren't needed for the queries, and I or whomever needs to look at which queries are actually used together to take those out. because its all macro stuff, the usual dead code detection doesnt tell us which. It's not harming anything other than maybe our compile times.

Reply:
I just did some poking at it now. Found a redundant one, the rest seem to be necessary. So i took out the comment entirely.